packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+- BugFix: Fixed the default version from "1.0" to "2.0" across job commands [#1459](https://github.com/zowe/zowe-cli/issues/1459)
```

Comment:
We should add a comment explaining how this may affect users as we discussed at standup 🙂 

The old default behavior was asynchronous - returned immediately after the request to cancel/delete the job was issued. The new behavior is synchronous - waits for confirmation from the server that the job has successfully been cancelled/deleted before returning.